src/core.c

```diff
@@ -35,7 +35,8 @@ u8 gNumHBlankIntrs = 0;
 struct BlendRegs gBldRegs ALIGNED(8) = {};
 u8 gOamFreeIndex = 0;
 struct Task gEmptyTask ALIGNED(16) = {};
-BgAffineReg gBgAffineRegs[NUM_AFFINE_BACKGROUNDS] ALIGNED(8) = {};
+u8 gNextFreeAffineIndex = 0;
```

Comment:
Ok, so I understand that including this means that we don't need to `ALIGNED(8)` the struct, but all structs are 8 aligned so I don't think the 8 align is because this var was actually here. Think I'd be happier with the `CURRENT_GAME` macro here too?

Reply:
I'd say it just means that there are more variables we need to uncover.
The space here didn't just appear at random.

Reply:
No, I don't think so. Structs are always 16 aligned and arrays are always 8 aligned. I'm ok with the extra if for accuracy